cpp/ql/src/Likely Bugs/Likely Typos/UsingStrcpyInConditional.qhelp

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>This rule finds uses of the string copy function calls that return the <code>destination</code> parameter, 
+and that do not have a return value reserved to indicate an error.</p>
+
+<p>The rule flags occurrences using such string copy functions as the conditional of an <code>if</code> statement, either directly, as part of an equality operator or a logical operator.</p>
+  
+<p>The string copy functions that the rule takes into consideration are: 
+<li>strcpy</li>
```

Comment:
I believe the QHelp failure is because these `<li>` elements aren't allowed inside a <`p`>. I think they need to be placed in a separate `<ul>` or `<ol>` block at the same level as the `<p>`; probably a `<ul>` in this case.